Kitodo-FileManagement/src/main/java/org/kitodo/filemanagement/FileManagement.java

```diff
@@ -157,7 +157,16 @@ public URI rename(URI uri, String newName) throws IOException {
             return null;
         }
 
-        String substring = uri.toString().substring(0, uri.toString().lastIndexOf('/') + 1);
+        String uriString = uri.toString();
+        if (isDirectory(uri)) {
+            if (uriString.equals(newName + "/") || uriString.equals(newName)) {
+                return null;
```

Comment:
Javadoc of `URI rename(URI uri, String newName)` in _org.kitodo.api.filemanagement.FileManagementInterface_ states:

    * @return true, if successful, false otherwise

Please update Javadoc in `FileManagementInterface`.

Reply:
Done